docs/rfc/2022-09-16-local-evaluation.md

```diff
@@ -0,0 +1,68 @@
+# MatrixOne Operator Local Evaluation
+
+Prerequisites
+
+- Docker
+
+## Components of image
+
+image based on `docker:22.06-rc-dind`
+
+- awscli
+- kubectl
+- kind
+- helm
+- helm repo (operator, minio)
+- modev (command line tools)
+
+## Start a matrixone cluster
+
+First, We should pull image from docker hub
+
+```shell
+docker pull matrixorigin/matrixone-operator:playground
+```
+
+Second, Start playground docker container
+
+```shell
+docker run -d --name playground --privileged -it -v ~/.aws:/root/.aws  matrixorigin/matrixone-operator:playground
+docker exec -it playground /bin/sh
+```
+
+Then, Start a matrixone cluster 
+
+```shell
+modev start
+```
```

Comment:
can the user connect to the database after one command?

e.g., after a docker run command, the cluster will be created (use default settings) and the sql interface is available